scripts/tests

```diff
@@ -0,0 +1,261 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: Apache-2.0
+
+import platform
+import sys
+import os
+import subprocess
+import logging
+import click
+import hashlib
+from functools import reduce
+from enum import Enum
+
+
+def config_logger(verbose):
+    if verbose:
+        logging.basicConfig(format="%(levelname)-5s > %(message)s", level=logging.DEBUG)
+    else:
+        logging.basicConfig(format="%(levelname)-5s > %(message)s", level=logging.INFO)
+
+
+def sha256sum(result):
+    m = hashlib.sha256()
+    m.update(result)
+    return m.hexdigest()
+
+
+def base_run(bin, force_qemu, verbose):
+    if force_qemu or (platform.system() == "Linux" and platform.machine() == "x86_64"):
+        logging.debug(f"Emulating {bin} with QEMU")
+
+        p = subprocess.run(
+            ["make", "CROSS_PREFIX=aarch64-none-linux-gnu-", f"{bin}"],
+            stdout=subprocess.DEVNULL if not verbose else None,
```

Comment:
One more thing: Can we please keep the output of `make` in the CI? If something goes wrong, it's good to know what exact commands were attempted.

Reply:
oh, yes good suggestion!